src/components/home/slider/style.tsx

```diff
@@ -88,6 +88,16 @@ export const ModalInfo = styled.div`
   box-shadow: 0 0px 12px rgba(0, 0, 0, 0.11), 0 10px 40px rgba(0, 0, 0, 0.19);
   border-top-right-radius: 8px;
   border-bottom-right-radius: 8px;
+
+  @media only screen and (min-width: 320px) {
+    display: none;
+  }
+  @media only screen and (min-width: 768px) {
+    display: flex;
+  }
+  @media only screen and (min-width: 1200px) {
+    display: flex;
+  }
```

Comment:
```suggestion
  @media only screen and (min-width: 768px) {
    display: flex;
  }
```

1 is enough for this case.